pkg/plugins/platform/local.go

```diff
@@ -416,10 +448,73 @@ func (l *LocalApp) ComposeFiles() []string {
 	return files
 }
 
+// ProcessHooks executes commands defined in a ddevapp.Command
+func (l *LocalApp) ProcessHooks(hookName string) error {
+	var err error
+
+	for _, c := range l.AppConfig.Commands[hookName] {
+		if c.ImportDB.Src != "" {
+			fmt.Printf("--- Importing database from %s ---\n", c.ImportDB.Src)
+			err = l.ImportDB(c.ImportDB.Src, c.ImportDB.ExtractPath)
+			if err != nil {
+				return fmt.Errorf("%s database import failed: %v", hookName, err)
+			}
+			util.Success("--- %s database import succeeded ---", hookName)
```

Comment:
I feel like this sort of messaging should be rolled into the actual import commands. Otherwise, we end up in a situation where output for the same operations varies depending on where/how it is triggered. 

I mostly say this because it bit me some on the pantheon work.